spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -683,6 +684,31 @@ def test_browse_history_dbg(ipyconsole, qtbot):
     qtbot.keyClick(control, Qt.Key_Up)
     assert '!aa = 10' in control.toPlainText()
 
+    # Open new widget
+    client_name = ipyconsole.get_current_client().get_name()
+    ipyconsole.create_new_client()
+    assert client_name != ipyconsole.get_current_client().get_name()
```

Comment:
This line is not needed.

Reply:
I want to make sure I am in a new tab. How should I do that?

Reply:
> ipyconsole.create_new_client()

That always create a new tab, so there's no need of that extra check.